docs/getting-started/my-account/README.md

```diff
@@ -122,32 +119,84 @@ Click the grey **Register device** button to register an authentication app. Ope
 
 Click the blue **Continue** button to finish the registration.
 
-![register authentication app](1572879640187.png)
+![openproject_my_account_authenticator_app](openproject_my_account_authenticator_app.png)
```

Comment:
@marcalcobe, i've asked Markus and as long as I do not intent to use it, it's fine. However, to be on the safer side, i've edited the image, so that the qr code leads to op documentation page (https://www.openproject.org/docs/).